dune

```diff
@@ -313,14 +312,17 @@
  (modes native)
  (instrumentation
   (backend bisect_ppx))
+ ; We don't use [unix] in [libraries] as that causes ocaml/otherlibs/unix/
+ ; to be built against the system compiler, which won't work.
+ (ocamlc_flags unix.cma)
+ (ocamlopt_flags unix.cmxa)
```

Comment:
I am a bit surprised we do not need `-I +unix`.

Reply:
Indeed, but everything seems to work.  I'll check again after all PRs are merged.